src/FuelClient.cc

```diff
@@ -1744,7 +1744,7 @@ bool FuelClientPrivate::FillModelForm(const std::string &_pathToModelDir,
     _form.emplace("owner", _owner);
   }
 
-  // \todo(nkoenig) The gz-fuelserver expects an integer number for the
+  // \TODO(nkoenig) The gz-fuelserver expects an integer number for the
```

Comment:
for style consistency, we're using lowercase `todo` notes. So not a typo here

```suggestion
  // \todo(nkoenig) The gz-fuelserver expects an integer number for the
```

Reply:
> for style consistency, we're using lowercase `todo` notes. So not a typo here

Hi @iche033, Thank you for pointing that out! I’ll work on the changes and update soon.